modules/cluster-management/src/testFixtures/java/org/apache/ignite/internal/cluster/management/raft/TestClusterStateStorage.java

```diff
@@ -62,62 +69,89 @@ public boolean isStarted() {
 
     @Override
     public void put(byte[] key, byte[] value) {
-        map.put(new ByteArray(key), value);
+        lock.readLock().lock();
```

Comment:
This is a modification operations, should write lock be taken?

Reply:
ooops, this is a typo